x/blobstream/client/verify.go

```diff
@@ -194,7 +198,7 @@ func sharesCmd() *cobra.Command {
 	return addVerifyFlags(command)
 }
 
-func VerifyShares(ctx context.Context, logger tmlog.Logger, config VerifyConfig, height uint64, startShare uint64, endShare uint64) (isCommittedTo bool, err error) {
+func VerifyShares(ctx context.Context, logger tmlog.Logger, config VerifyConfig, height int64, startShare uint64, endShare uint64) (isCommittedTo bool, err error) {
```

Comment:
this is also breaking

Reply:
this change is for consistency, right?

Reply:
Mostly motivated by consistency. If this was reverted to `uint64` then we'd have to do bounds checks / converting inside this function body.

- `trpc.Block` accepts a `*int64 height`
- `VerifyDataRootInclusion` accepts a `int64 height`.